docs/install/README.md

```diff
@@ -371,38 +373,451 @@ Status:
+
+Make the active context `kind-k8ssandra-0`:
+
+```console
+kubectx kind-k8ssandra-0
+```
+
+Restart the operator:
+
+```console
+kubectl -n k8ssandra-operator rollout restart deployment k8ssandra-operator
+```
+
+#### Deploy a K8ssandraCluster
+Now we will create a K8ssandraCluster that consists of a Cassandra cluster with 2 DCs and 3 
+nodes per DC, and a Stargate node per DC.
+
+```sh
+cat <<EOF | kubectl apply -f -
+apiVersion: k8ssandra.io/v1alpha1
+kind: K8ssandraCluster
+metadata:
+  name: demo
+spec:
+  cassandra:
+    cluster: demo
+    serverVersion: "3.11.11"
+    storageConfig:
+      cassandraDataVolumeClaimSpec:
+        storageClassName: standard
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 5Gi
+    config:
+      jvmOptions:
+        heapSize: 512M
+    networking:
+      hostNetwork: true    
+    datacenters:
+      - metadata:
+          name: dc1
+        size: 3
+        stargate:
+          size: 1
+          heapSize: 256M
+      - metadata:
+          name: dc2
+        k8sContext: kind-k8ssandra-1
+        size: 3
+        stargate:
+          size: 1
+          heapSize: 256M 
+EOF
+```
+
+## Kustomize
+
+### Single Cluster
+We will first look at a single cluster install to demonstrate that while K8ssandra 
+Operator is designed for multi-cluster use, it can be used in a single cluster without 
+any extra configuration.
+
+#### Create kind cluster
 Run `setup-kind-multicluster.sh` as follows:
 
 ```sh
 ./setup-kind-multicluster.sh --kind-worker-nodes 4
 ```
 
-### Install Cert Manager
+#### Install Cert Manager
 We need to first install Cert Manager as it is a dependency of cass-operator:
 
 ```console
 kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
 ```
 
-### Install K8ssandra Operator
-The GitHub Actions for the project are configured to build and push a new operator image to Docker Hub whenever commits are pushed to `main`. 
+#### Install K8ssandra Operator
+The GitHub Actions for the project are configured to build and push a new operator image 
+to Docker Hub whenever commits are pushed to `main`. 
 
-See [here](https://hub.docker.com/repository/docker/k8ssandra/k8ssandra-operator/tags?page=1&ordering=last_updated) on Docker Hub for a list of available images.
+See [here](https://hub.docker.com/repository/docker/k8ssandra/k8ssandra-operator/tags?page=1&ordering=last_updated) 
+on Docker Hub for a list of available images.
 
-If you wish to install the default installation, one can install with kubectl:
+Install with kubectl:
 
 ```console
 kubectl apply -k github.com/k8ssandra/k8ssandra-operator/config/deployments/default
 ```
 
-**Note:** This will deploy the `latest` operator image, i.e., `k8ssandra/k8ssandra-operator:latest`. In general it is best to avoid using `latest`. 
+This installs the operator in the `k8ssandra-operator` namespace.
```

Comment:
The line above isn't accurate anymore as it should be `deployments/control-plane` instead of `deployments/default`. Also, isn't the `https://` required in front?
I'm not able to run this, whatever I try actually. Is there a kubectl plugin that's necessary to get the -k flag to work?

I went with `kustomize build config/deployments/control-plane | kubectl apply -f -` to get this passing.

Reply:
kubectl 1.21+ should work fine with -k (it was on older versions also, but they changed update process in 1.21 which now includes newer kustomize always)

Reply:
>isn't the https:// required in front?

No